imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -76,6 +93,23 @@ func TestGuestSecureBoot(t *testing.T) {
 	}
 }
 
+func testWindowsGuestSecureBoot(t *testing.T) {
+	cmd := exec.Command("powershell.exe", "Confirm-SecureBootUEFI")
+
+	output, err := cmd.Output(); 
+	
+	if err != nil {
+		t.Fatalf("failed to run SecureBoot command: %v", err)
+	}
+
+	// The output will return a string that is either 'True' or 'False'
+	// so we need to parse it and compare here.
+	trimmed_output := strings.TrimSpace(string(output))
+	if trimmed_output != "True" {
+		t.Fatalf("Secure boot is not enabled as expected: %v", trimmed_output)
```

Comment:
won't this be "Secure boot is not enabled as expected: False" in all cases? is it useful to include?

Reply:
Ah, you're right. Fixed.